File: java_basics/oops_example.java


public class oops_example {
    
String brand;
Integer year;

public oops_example(String brand, Integer year)
{
    this.brand=brand;
    this.year=year;
}

public Integer get_age(Integer current_year)
{
   return current_year - this.year;
}

}

File: python_basics/oops_example.py


class Car:

    def __init__(self, brand, year):
        self.brand = brand
        self.year=year

    def get_age(self,current_year):
        return current_year-self.year


toyota=Car("Toyota", 2018)
print(toyota.brand)
print(toyota.year)
print(toyota.get_age(2026))    